instrumentation/opentelemetry-instrumentation-elasticsearch/src/opentelemetry/instrumentation/elasticsearch/__init__.py

```diff
@@ -97,17 +97,23 @@ def _instrument(self, **kwargs):
         """
         tracer_provider = kwargs.get("tracer_provider")
```

Comment:
Can you update the usage docs with an example of using the request/response hooks with their function signatures?

Reply:
done.

Reply:
Done.